test/chat_models/chat_vertex_ai_test.exs

```diff
@@ -456,4 +505,102 @@ defmodule ChatModels.ChatVertexAITest do
       assert inspect(changeset) =~ "**redacted**"
     end
   end
+
+  describe "live tests and token usage information" do
+    @tag live_call: true, live_vertex_ai: true
+    test "basic non-streamed response works and fires token usage callback" do
+      handlers = %{
+        on_llm_token_usage: fn usage ->
+          send(self(), {:fired_token_usage, usage})
+        end
+      }
+
+      chat =
+        ChatVertexAI.new!(%{
+          model: "gemini-2.5-flash",
+          temperature: 0,
+          endpoint:
+            "https://aiplatform.googleapis.com/v1/projects/eng-sandbox-7s/locations/global/publishers/google",
```

Comment:
Is this endpoint something you want to include? It looks like it might be specific to you? If so, this could pull in an ENV for the endpoint. If there's a standard API endpoint available for use, it could be that one.